Site/ASAB Maestro/Descriptors/asab-config.yaml

```diff
@@ -14,6 +14,18 @@ descriptor:
   # This is critical service
   restart: always
 
+# Other services can assign content through asab-config section in the descriptors. This sherpa uploads all content to asab-config.
+sherpas:
+  content:
+    image: docker.teskalabs.com/asab/asab-governator:stable
+    volumes:
+      - "{{SITE}}/{{INSTANCE_ID}}/sherpa:/sherpa"
+    command: python3 /sherpa/script.py
```

Comment:
Just a comment: I find a bit less useful to name the directory of "sherpa" being "sherpa" ... it is technical naming that doesn't provide any hint about the actual functionality.

The name should say "WHAT it does" instead "WHAT is it